docs/directives/needuml.rst

```diff
@@ -152,6 +152,39 @@ and can't be ``diagram``.
       B -> C: Hi
       C -> B: Hi there
 
+save
+~~~~
+
+Allows to store generated Plantuml-code of ``needuml`` to given file path. The given file path must be
+relative path to ``conf.py`` directory. For example, ``_build/_out/my_needuml.puml``.
```

Comment:
Is this still valid?

I thought it is relative to the current builder output folder, e.g _build/needumls